src/main/java/seedu/address/model/event/Event.java

```diff
@@ -57,15 +76,26 @@ public boolean equals(Object other) {
 
         Event otherEvent = (Event) other;
         return otherEvent.getDescription().equals(getDescription())
-                && otherEvent.getTime().equals(getTime());
+                && otherEvent.getTime().equals(getTime())
+                && otherEvent.getAssociatedPersons().equals(getAssociatedPersons());
     }
 
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getDescription())
-                .append(" At: ")
+                .append("\nAt: ")
                 .append(getTime());
+
+        if (associatedPersons.size() != 0) {
+            builder.append("\n\nPeople attending: ");
```

Comment:
I believe this is a typo unless you mean 2 empty lines

Reply:
Its next line followed by skip 1 line. Example:
*info*
At: 12-12-1234 12:34

People attending: 
*info*